fabric/src/main/java/dev/latvian/mods/kubejs/fabric/KubeJSFabric.java

```diff
@@ -23,7 +25,20 @@ public void onInitialize() {
 	}
 
 	public void registerObjects() {
+		var ignored = new HashSet<>(RegistryInfo.AFTER_VANILLA);
+
 		for (var info : RegistryInfo.MAP.values()) {
+			if (!ignored.contains(info)) {
+				var deferredRegister = DeferredRegister.create(KubeJS.MOD_ID, UtilsJS.cast(info.key));
+				int added = info.registerObjects(deferredRegister::register);
+
+				if (added > 0) {
+					deferredRegister.register();
+				}
+			}
+		}
+
+		for (var info : RegistryInfo.AFTER_VANILLA) {
 			var deferredRegister = DeferredRegister.create(KubeJS.MOD_ID, UtilsJS.cast(info.key));
 			int added = info.registerObjects(deferredRegister::register);
 
```

Comment:
Any reason in particular we're using DeferredRegister here? On Fabric, DeferredRegister just registers directly to the registry iirc, so we could just replace it with the appropriate call immediately